tests/test_runtime_errors.py

```diff
@@ -35,21 +35,33 @@ def test_missing_command() -> None:
         main(["some-fictional-command", "tests", "--some-flag"])
 
 
-def test_missing_root_dir() -> None:
+def test_missing_root_dir(capsys) -> None:
     """Check useful error message is raised if :code:`nbqa` is called without root_dir."""
     msg = dedent(
         """\
-        \x1b\\[1;31mPlease specify both a command and a notebook/directory\x1b\\[0m, e.g.:
+        usage: nbqa <code quality tool> <notebook or directory> <flags>
```

Comment:
Would it be better to have `nbqa <code quality tool> <notebook or directory> <nbqa options> <code quality tool arguments>`